src/components/StatusPill/StatusPill.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+import { StatusDot } from '../StatusDot/index';
+import { StatusPillProps } from './StatusPill.types';
+import PillLabel from '../Pill/PillLabel';
+import { SpaceSizes } from '../../theme';
+import { Inline, Padbox } from '../layout';
+import { PaddingTypes } from '../layout/Padbox/Padbox.enums';
+import { getColor, getRadii, pxToRem } from '../../utils';
+
+const StyledPillWrapper = Styled(Padbox)`
+  display: inline-block;
+  height: ${pxToRem(24)};
+  min-width: 0;
+  border-radius: ${getRadii('round')};
+  border: 1px solid ${getColor('neutral.200')};
+  background-color: ${getColor('neutral.0')};
+  padding-right: ${pxToRem(6)};
+  padding-top: ${pxToRem(3)};
```

Comment:
I understand why the right padding is different, but why the top is set to 3?

Reply:
padding xs squish make the component 22 px, square 26, fix size is ok but content is 1 px up